### DISCUSSION TITLE
ReVanced progress update

### DISCUSSION THREAD

**Discussion by oSumAtrIX** [ReVanced/revanced-discussions#176] [Announcements]
# 💊 Dalvik patcher & patches

The Dalvik patcher and patches branches have been finally merged to the repository's respective dev branches 🎉
The focus of development can now slowly shift towards extending the number of patches and keeping them up to date.

# 💻 ReVanced CLI

The picoli branch is now merged into the main branch allowing it to work with the latest patches and patcher.
Additionally, packages are now being published and can be downloaded.

# 📝 Documentation

The documentation has been updated to match the latest breaking changes.

**Comment by Made0fB1naries** [comment]
👀

**Comment by Alchemytr** [comment]
Congratulations to all the  
hard-working talented people who contributed (and continue to contribute) to this project.
